components/src/components/SimpleDateTime/Component.ts

```diff
@@ -1,62 +1,74 @@
 /* tslint:disable */
-import { Components } from 'formiojs';
+import _ from "lodash";
+import { Components } from "formiojs";
 const ParentComponent = (Components as any).components.datetime;
-import editForm from './Component.form';
+import editForm from "./Component.form";
```

Comment:
Not sure why linting didn't pick up on this, but strings are supposed to be single quoted, not double quoted. Looks like the latest refactor changed everything to double quotes for some reason.

Reply:
Fixed all Linting errors in next commit.

Reply:
I believe that my formatter for VSCode was introducing these Linting errors. It switched to Prettier for some reason, so I switched it back to the original VSCode formatter.